src/components/App/App.js

```diff
@@ -48,21 +48,16 @@ export default function App() {
     filterByType(type)
 	}
 
-  function toDollars(number) {
-    return `{$}number.toString()`
-  };
-
   function findByName(name) {
     const usersWatchByName = watches.find(watch => {
        return watch.name === name;
     });
     return setWatchByName(usersWatchByName);
    }
-   console.log(watchByName)
 
   function filterByPrice(price) {
     const userswatchesByPrice = watches.filter(watch => {
-      return watch.price === toDollars(price);
```

Comment:
If i understand this, you've combined the functionality of 2 functions, filterbyPrice and toDollars, into one functions. If that the case, this is sick! I like how you use filter to sort out the 2 pieces of information.